loopd/swapclient_server.go

```diff
@@ -483,26 +483,41 @@ func (s *swapClientServer) GetLoopInQuote(ctx context.Context,
 		return nil, err
 	}
 
+	var routeHints [][]zpay32.HopHint
+	haveRouteHints := len(req.LoopInRouteHints) != 0
+	haveLastHop := req.LoopInLastHop != nil
 	var lastHop *route.Vertex
-	if req.LoopInLastHop != nil {
+
+	switch {
```

Comment:
nit: for `switch` we have a code style we use all across the codebase: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#code-spacing